tools/downloader/README.md

```diff
@@ -199,7 +200,7 @@ The basic usage is to run the script like this:
 ```
 
 This will convert all models into the Inference Engine IR format. Models that
-were originally in that format are ignored. Models in PyTorch's format will be
+were originally in that format are ignored. Models in PyTorch's and Caffe2 format will be 
```

Comment:
Inconsistent grammar: "PyTorch's and Caffe2". Also, please don't add trailing whitespace.